wavelength-plugins/splatoon.js

```diff
@@ -0,0 +1,192 @@
+/****************************
+ * Splatoon Plug-in for PS	*
+ * Created by Insist		*
+ ****************************/
+
+"use strict";
+
+let SPLATFEST = false;
+let splatfestTeams = [];
+
+exports.commands = {
+	"spl2n": "splatoon",
+	"splatoon2": "splatoon",
+	splatoon: {
+		ranking: "rank",
+		ranks: "rank",
+		rank: {
+			clamblitz: "cb",
+			cb: function (target, room, user) {
+				if (!target) return this.parse("/splatoonhelp");
+				if (!["c-", "c", "c+", "b-", "b", "b+", "a-", "a", "a+", "s", "s+"].includes(target.toLowerCase())) return this.errorReply(`Invalid Ranking; check your spelling?`);
```

Comment:
Use the proper case for a rank ex: `C-`, `B`, `S+`.
Also set the target to the proper case before testing so when its written to the DB its done right.
`target = target.trim().toUpperCase();`
Do this for all 4 commands.